Content.Server/Atmos/Piping/Other/EntitySystems/GasMinerSystem.cs

```diff
@@ -25,10 +25,17 @@ private void OnMinerUpdated(Entity<GasMinerComponent> ent, ref AtmosDeviceUpdate
         {
             var miner = ent.Comp;
 
+            if (!GetValidEnvironment(ent, out var environment))
+            {
+                miner.Idle = false;
```

Comment:
Should this set idle to true or false?

Reply:
this is why we do code review 😅

Reply:
I believe this catches all of them. Though, since we are already changing the name of the variable, do you think that changing it to the affirmative action (e.g. `miner.Mining`) would be better than the negative action (`miner.Idle`)? If you think `Idle` is better, we can certainly keep that.

Reply:
lets keep it as-is